src/main/java/solutions/fairdata/fdp/index/api/controller/ApiExceptionHandler.java

```diff
@@ -0,0 +1,17 @@
+package solutions.fairdata.fdp.index.api.controller;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import solutions.fairdata.fdp.index.api.dto.ErrorDTO;
+import solutions.fairdata.fdp.index.exceptions.IncorrectPingFormatException;
+
+@ControllerAdvice
+public class ApiExceptionHandler {
+
+    @ExceptionHandler(IncorrectPingFormatException.class)
+    public ResponseEntity<ErrorDTO> handleIndexException(IncorrectPingFormatException exception) {
+        return new ResponseEntity<>(exception.getErrorDTO(), HttpStatus.BAD_REQUEST);
```

Comment:
Could use the `getCode()` method instead of hardcoding the `HttpStatus` value (either through `resolve` or `valueOf`, depending on how strict it needs to be).